### DISCUSSION TITLE
The sticky Chapter menu needs "more content" above/below indicators

### DISCUSSION THREAD

**Discussion by richardkmichael** [rails/rails#52526] [Docs]
@AmandaPerino 

The new chapter side menu looks very nice!  ✨  I like the scroll position indicator. 👏 

Two suggestions:

1. The menu can display content such that it's impossible to tell it is scrollable, and there is additional content.  This can occur either above or below, but below is the obvious case -- when a page first loads, there can be no indication there is more to read.  This is a particularly bad on macOS which hides the scroll bar.  I think this probably will most affect new users, who simply won't know there is more to know. 📖 🧑‍🎓 

2. I think the chapter menu could be slightly taller.  Several guides have lots of content, just showing a bit more would reduce user-scrolling.

<img width="648" alt="chapter-menu-no-scroll-indicator" src="https://github.com/user-attachments/assets/963c51c3-d084-44da-accd-0dfe8f815f8d">

**Comment by AmandaPerino** [comment]
Hi @richardkmichael! What browser are you using? I am on Chrome (on mac) and the scroll bar is always visible, but we may have overlooked a browser when working on this.

<img width="440" alt="Screenshot 2024-08-12 at 21 34 40" src="https://github.com/user-attachments/assets/d1433a5d-d547-4dd5-9027-fb956b4e69b2">

**Comment by richardkmichael** [comment]
Hello!

I'm using Firefox 129.

I just checked Chrome, and I do see the scroll bar as in your screenshot, so I guess it is an issue with Firefox.
